[04-Dec-2023 12:30:40 America/Sao_Paulo] PHP Fatal error:  Uncaught mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: NO) in /home1/cyber065/public_html/cardapio/PHP/conn.php:3
Stack trace:
#0 /home1/cyber065/public_html/cardapio/PHP/conn.php(3): mysqli->__construct('localhost', 'root', Object(SensitiveParameterValue), 'cyber')
#1 /home1/cyber065/public_html/cardapio/PHP/chamar.php(2): include_once('/home1/cyber065...')
#2 {main}
  thrown in /home1/cyber065/public_html/cardapio/PHP/conn.php on line 3
